dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/constants/CoreConstants.java

```diff
@@ -155,6 +155,7 @@ private CoreConstants() {}
 			VIEW_MONITOR = "monitor",
 			VIEW_OVERVIEW = "overview",
 			PERF_MONITOR = "performance/charts",
+			PERF_METRICS_DASH = "performance/codewind-metrics/dashboard",
```

Comment:
I have changed the path to the new dashboard, like so:
```suggestion
			PERF_METRICS_DASH = "performance/monitor/dashboard",
```